wire-compiler/src/test/java/com/squareup/wire/WireCompilerTest.java

```diff
@@ -90,7 +90,9 @@ private void testProto(String[] sources, String[] outputs, String serviceWriter,
     invokeCompiler(args);
 
     List<String> filesAfter = getAllFiles(testDir);
-    assertEquals(filesAfter.toString(), outputs.length, filesAfter.size());
+    if (outputs.length != filesAfter.size()) {
```

Comment:
That snuck in by mistake. (I wanted to hit a breakpoint when the values were different.)